Ports/ruby/package.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+source version.sh
+
+port=ruby
+version="${RUBY_VERSION}"
```

Comment:
Not sure why this is an env variable, if you the patches are for version 3.0.2 of ruby this should specify that explicitly.

Reply:
I started from copying the python port which does the same thing. I can hard-code it instead if desired.

Reply:
the python port only does that because cross compiling python requires the same minor version on the host, so version information is shared between the port and a toolchain build script.
please don't do that here if not needed.

Reply:
doh... I totally forgot to mention that actually ruby _does_ require the same version on the build host when cross compiling.